### DISCUSSION TITLE
data normalization before processing AutoML?

### DISCUSSION THREAD

**Discussion by Tonywhitemin** [mljar/mljar-supervised#572] [General]
Hi @pplonski 
Wish you have a great day!
As title, I'd like to check if our data will do normalization before processing AutoML or not?
Thanks for your help!

**Comment by pplonski** [comment]
Hey. Its up to you if you would like to preprocess data before AutoML. The AutoML will have less work if data is prepared. Is that your question?

**Reply by Tonywhitemin** [reply]
@pplonski 
Oh... sorry for my mistake...
I mean if mljar will automatically do data normalization or not?
If yes, what method will it used?(min-max or?)
Thank you!

**Comment by pplonski** [comment]
Yes it will automatically normalize data to zero mean and 1 std.

**Reply by Tonywhitemin** [reply]
Thank you @pplonski !
But I found that if the algorithms are tree based (like decision tree, Xgboost),  it won't use normalization method, is that correct? 
Please refer to following picture is from the framework.json in "Linear" folder:
![Screenshot from 2022-10-06 13-53-44](https://user-images.githubusercontent.com/88414200/194226232-3a6ad075-b3c9-461d-bfe0-42429ce83fdc.png)

And below is from the framework.json in "DecisionTree" folder: (from same input training file)
![Screenshot from 2022-10-06 13-59-20](https://user-images.githubusercontent.com/88414200/194226245-d11b9a37-3c44-48f1-b7a3-f0841feaec36.png)

**Comment by pplonski** [comment]
Thats correct. Tree based methods care only about ordering, scaling is not needed.
